meshtastic/node_runtime/response_runtime.py

```
@@ -0,0 +1,378 @@
+                )
+                self._channel_request_failed = True
+                self._node._timeout.reset()  # noqa: SLF001
+            elif error_reason != "NONE":
+                self._retry_pending_channel_request(error_reason)
+            else:
+                logger.debug(
+                    "Channel request routed successfully; waiting for ADMIN_APP payload."
+                )
+        return True
+
+    def handle_channel_response(
```

Ruff (0.15.6)

[warning] Too many return statements (9 > 6)

(PLR0911)